where-to-travel/src/main/webapp/script.js

```diff
@@ -259,13 +263,14 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
 
     function callback(response, status) {    
       if (status == 'OK') {
-        var origins = response.originAddresses;
-        var destinations = response.destinationAddresses;
-      
-        for (var i = 0; i < origins.length; i++) {
-          var results = response.rows[i].elements;
-          for (var j = 0; j < results.length; j++) {
-            var element = results[j];
+        const origins = response.originAddresses;
```

Comment:
@emmapaczkowski In javascript there is some pretty nice syntactic sugar for looping through an array. If you want, you could replace
line 269 with for(row of response.rows) and line 272 with for(result of results).

Reply:
done